docs/src/main/sphinx/admin/properties-logging.rst

```diff
@@ -2,14 +2,36 @@
 Logging properties
 ==================
 
+``log.annotation-file``
+^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-string`
+* **Default format:** ``JSON``
+
+An optional file that allows a file containing fields to be set at startup. It
```

Comment:
```
An optional properties file that contains annotations to be included with
each log message. This can be used to include machine-specific or
environment-specific information into logs which are centrally aggregated.
The annotation values can contain references to environment variables.

.. code-block:: properties

    environment=production
    host=${ENV:HOSTNAME}